src/Symfony/Component/Messenger/Bridge/Amqp/Transport/Connection.php

```diff
@@ -324,6 +326,10 @@ private function publishOnExchange(\AMQPExchange $exchange, string $body, string
             $amqpStamp ? $amqpStamp->getFlags() : AMQP_NOPARAM,
             $attributes
         );
+
+        if (isset($this->connectionOptions['confirm_timeout'])) {
```

Comment:
`'' !== ($this->connectionOptions['confirm_timeout'] ?? '')` to be compatible with the DSN syntax